src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.h

```diff
@@ -131,7 +131,7 @@ calls with @ref InputFileCallbackPolicy::LoadTemporary and
 @ref InputFileCallbackPolicy::Close is emitted right after the file is fully
 read.
 
-Import of skeleton, skin and morph data is not supported at the moment.
+Import of skeleton and morph data is not supported at the moment.
```

Comment:
I removed the "skeleton" from here as well and merged as f9e41c62ad43c8b314bfad7180d413032fdb5d15. This was originally here only because I had very little idea about what's the difference between a skin and a skeleton, and given that it's just a single ID that (apparently) almost nobody uses, I don't think it needs to be mentioned here.

Thanks for the PR! :+1: